Spigot-Server-Patches/0402-Only-count-Natural-Spawned-mobs-towards-natural-spaw.patch

```diff
@@ -0,0 +1,60 @@
+From 13cdbbfbad353171b8f8b20f1d9785f24246de96 Mon Sep 17 00:00:00 2001
+From: kickash32 <kickash32@gmail.com>
```

Comment:
Please restore original patch headers with me as author and original date.

Reply:
Is this good?

Reply:
Missing original date too.